Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml

```diff
@@ -3,6 +3,8 @@
   startingInventory:
     Handcuffs: 8
     GrenadeFlashBang: 4
+    ClusterBangFull: 4
+    GrenadeStinger: 8
```

Comment:
these could be toned down a little bit. there's quite a few in here.

Reply:
Halved both